cmd/infracost/run.go

```diff
@@ -233,12 +233,7 @@ func runProjectConfig(cmd *cobra.Command, runCtx *config.RunContext, ctx *config
 	provider, err := providers.Detect(ctx)
 	if err != nil {
 		m := fmt.Sprintf("%s\n\n", err)
-		m += fmt.Sprintf("Use the %s flag to specify the path to one of the following:\n", ui.PrimaryString("--path"))
-		m += " - Terraform plan JSON file\n - Terraform/Terragrunt directory\n - Terraform plan file"
-
-		if cmd.Name() != "diff" {
-			m += "\n - Terraform state JSON file"
-		}
+		m += fmt.Sprintf("Try setting --path to a Terraform plan JSON file. See %s for how to generate this.", ui.LinkString("https://infracost.io/troubleshoot"))
```

Comment:
Going with /troubleshoot just now. I think it'll be okay to change it in future releases if we update the URL